tests/Feature/ValidateCSVTest.php

```diff
@@ -33,7 +33,7 @@ public function invalidLineProvider(): iterable
         yield 'missing statement guid' => [
             function (array $config): array {
                 return [
-                    ',P569,3 April 1934,1934-04-03,http://www.example.com', // Emulate missing guid
+                    ',P569,3 April 1934,,1934-04-03,http://www.example.com', // Emulate missing guid
```

Comment:
Do we have some tests that leave out the new field? IIUC, it’s supposed to be optional – not only can you leave the column empty in your CSV, but not having the column at all is also supposed to be supported, I believe.

Reply:
I tried it out and it looks like the new column is actually required:

> CSV import parsing error at line 0: Unrecognized column headers, please include exactly the following column headers: statement_guid, property_id, wikidata_value, meta_wikidata_value, external_value, external_url